UnityProject/Assets/Scripts/Managers/CentComm.cs

```diff
@@ -314,7 +320,8 @@ public static void MakeShuttleRecallAnnouncement( string text )
 	{
 		Chat.AddSystemMsgToChat(string.Format( PriorityAnnouncementTemplate, string.Format(ShuttleRecallSubTemplate,text) ),
 			MatrixManager.MainStationMatrix);
-		SoundManager.PlayNetworked("ShuttleRecalled");
+		// JESTER
+		//SoundManager.PlayNetworked("ShuttleRecalled");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)